formats/metadata.json

```diff
@@ -11,9 +10,6 @@
     "locale"
   ],
   "properties": {
-    "base_path": {
-      "type": "string"
-    },
```

Comment:
maybe i'm missing something, but if metadata.json is supposed to be the master definition, and shared across publishers and frontend [^1], shouldn't we be preserving `base_path` here? at least as an optional field?

[1] https://github.com/alphagov/govuk-content-schemas#publisher-schema-defined-using-component-parts